### DISCUSSION TITLE
tester for docker wanted

### DISCUSSION THREAD

**Discussion by superkalt** [frangoteam/FUXA#397] [Show and tell]
hi community,

I am looking for a tester for the docker container.
additionally I created a script to start the whole thing quickly on a synology nas.

[https://github.com/superkalt/FUXA](https://github.com/superkalt/FUXA)

**Comment by abdel2010** [comment]
Hello,

Are you still looking for a docker tester?

**Reply by MatthewReed303** [reply]
docker is working fine. I'm going to look into setting up docker-compose and see if that works ok ( don't see why not ) 
are you having trouble?

**Reply by abdel2010** [reply]
Hello, thanks for replying.

I have a an issue exposing ports for communication protocls (OPCUA and MQTT for example)

**Reply by MatthewReed303** [reply]
are you trying to expose them or publish them? You need to add the ports to the docker run -p 4840:4840 and -p 1883:1883

or you can define them to anything, but just remember the left side is your host and right is docker container ports
